.github/workflows/ci.yml

```diff
@@ -90,3 +90,36 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           unchecked: '1,5069'
+  
+  check-license-lines:
+    name: Check licenses
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+      
+      - name: Get Changed Files
+        id: changed-files
+        uses: tj-actions/changed-files@6c44eb8294bb9c93d6118427f4ff8404b695e1d7
+
+      - name: Delete Unmodified Files
+        run: |
+          cd $GITHUB_WORKSPACE
+          mkdir all
+          shopt -s extglob dotglob
+          mv !(all) all
+          shopt -u dotglob
+          mkdir unchanged
+          cd all
+          for FILE in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            [ -f ${FILE} ] && mkdir -p $(dirname "../unchanged/${FILE}")
+            [ -f ${FILE} ] && cp ${FILE} ../unchanged/${FILE}
+          done
+          cd ..
+          rm -rf all
+          cp -r unchanged/* .
+          cd ..
+          rm -rf unchanged
```

Comment:
While not a hard blocker, I'm not a huge fan of obscure uncommented shell scripts.  Is there any way to *easily* turn this into something that is more readable like a python or TS script (or perhaps a GitHub action for "remove all unchanged files"?

Reply:
Alternatively, some comments may help make this block more understandable... like I'm pretty sure at the moment that `shopt -u dotglob` was a cat walking across your keyboard while you were in the middle of authoring this.

Reply:
Yea, I'm planning on making it an action. This is a nice stop-gap.

Reply:
And no, `shopt -u dotglob` is not due to an unherded Ethereum cat.

Reply:
I have been thinking about this more, and I think I preferred the file ignore-list approach as it was overall simpler than this one, which requires an extra two steps, one of which is an external dependency and one of which is an obscure script.  In general, I really like simple solutions as our CI already has enough problems as it is due largely (I suspect) to its complexity.  If you were able to create a single action that handled all of this I would be more compelled to keep the pattern as it is, but 4 actions just to check for the presence of first line of files feels like overkill on complexity.